openedx/adg/lms/webinar/models.py

```diff
@@ -0,0 +1,73 @@
+"""
+All models for webinar app
+"""
+from django.contrib.auth.models import User
+from django.core.validators import FileExtensionValidator
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from model_utils.models import TimeStampedModel
+
+from .constants import ALLOWED_BANNER_EXTENSIONS
+
+
+class Webinar(TimeStampedModel):
+    """
+    Model to create a webinar
+    """
+
+    start_time = models.DateTimeField(verbose_name=_('Start Time'), )
+    end_time = models.DateTimeField(verbose_name=_('End Time'), )
+
+    subject = models.CharField(verbose_name=_('Subject'), max_length=255, )
+    title = models.CharField(verbose_name=_('Title'), max_length=150, )
+    description = models.TextField(verbose_name=_('Description'), )
+    presenter = models.ForeignKey(
+        User, verbose_name=_('Presenter'), on_delete=models.CASCADE, related_name='webinar_presenter',
+    )
+    meeting_link = models.URLField(default='', verbose_name=_('Meeting Link'), blank=True, )
+    banner = models.ImageField(
+        upload_to='webinar/banners/',
+        verbose_name=_('Banner'),
+        validators=[FileExtensionValidator(ALLOWED_BANNER_EXTENSIONS)],
+        help_text=_('Accepted extensions: .png, .jpg, .svg'),
+    )
+    language = models.CharField(verbose_name=_('Language'), max_length=150, )
+    co_hosts = models.ManyToManyField(User, verbose_name=_('Co-Hosts'), blank=True, related_name='webinar_co_hosts', )
+    panelists = models.ManyToManyField(
+        User, verbose_name=_('Panelists'), blank=True, related_name='webinar_panelists',
+    )
+    registered_users = models.ManyToManyField(
+        User, verbose_name=_('Registered Users'), blank=True, related_name='webinar_registered_users',
+    )
```

Comment:
Lets discuss this in a call. I think we should go for a separate model so that we can store user related thing in future. Like whether they cancelled or something like that.